hw2/03-count.html

```diff
@@ -6,19 +6,20 @@
     <title>03 - Count</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="03-count.js"></script>
   </head>
   <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
+    <main class="container center-box">
+      <h1 class="container mt-4 text-center">Exercise 03 - Count</h1>
       <hr />
       <div>
         <label>Highlight all occurences of a word:</label>
-        <input />
+        <input id="userInput" />
```

Comment:
WAVE extension flagged as input doesn't have a label and the label is not associated with an input

Reply:
This page also needs to be styled